src/functions/src/submitContactForm.ts

```diff
@@ -1,10 +1,46 @@
 import axios from 'axios';
 import * as admin from 'firebase-admin';
 import * as functions from 'firebase-functions';
+import { ApolloClient, createHttpLink, gql, InMemoryCache } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+// import { ApolloClient, createHttpLink, InMemoryCache, gql } from "@apollo/client";
+// import { setContext } from '@apollo/client/link/context';
 
 if (admin.apps.length === 0) {
   admin.initializeApp();
 }
+export async function createDiscussion(topic: string, body: string) {
+  const httpLink = createHttpLink({
+    uri: 'https://api.github.com/graphql',
+  });
+
+  const authLink = setContext((_, { headers }) => {
+    return {
+      headers: {
+        ...headers,
+        authorization: `Bearer ${process.env.GITHUB_ACCESS_TOKEN}`,
```

Comment:
what's `process.env.GITHUB_ACCESS_TOKEN`? i think the token that should be used is the key from firebase functions (`const key = functions.config().contactform.issueapikey;`), but lmk if the token's different